src/docs/tutorial.tex

```diff
@@ -459,11 +459,13 @@ \section{Booleans: {\tt bool}}
 boolean value is true:
 
 \begin{minted}{felix}
-assert$ 1 < 2;
+assert 1 < 2;
 \end{minted}
 
 If the argument of an \verb%assert% is false, then if control
 flows through it, the program is terminated with an error message.
+Note that as far as \verb%assert% is neither procedure nor function
```

Comment:
> On 29 Mar 2016, at 01:41, Ryan Gonzalez <notifications@github.com> wrote:
> 
> In src/docs/tutorial.tex:
> 
> >  \end{minted}
> >  
> >  If the argument of an \verb%assert% is false, then if control
> >  flows through it, the program is terminated with an error message.
> > +Note that as far as \verb%assert% is neither procedure nor function
> 
> This reads oddly to me. How about:
> 
> Note that, because \verb%assert% is a keyword, not a procedure, you do not need to fix operator priorities with \verb%$%.

Technically this is not correct. Felix doesn’t have any keywords .. 
well not many. “assert” is recognised as special in certain contexts
dictated by the grammar. In other contexts it is an ordinary identifier.

For example this program compiles and runs correctly:

var assert = 1;
assert assert == 1;
println$ assert == 1;


—
john skaller
skaller@users.sourceforge.net
http://felix-lang.org